src/core/map/popover/MapPopover.fixture.tsx

```
@@ -0,0 +1,301 @@
+        if (!map) return null;
+
+        const features =
+          mapEvent.target
+            ?.queryRenderedFeatures?.(mapEvent.point)
+            ?.filter((f) => f.source === 'hot-project-layers') || [];
+        if (features.length === 0) return null;
+
+        const feature = features[0];
+        if (!feature || !feature.properties) return null;
+
+        const data = hotData.find((d) => d.projectId === feature.properties!.projectId);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)